pydis_site/apps/api/tests/test_users.py

```diff
@@ -455,14 +458,35 @@ def test_metricity_voice_banned(self):
                     self.assertEqual(response.status_code, 200)
                     self.assertEqual(response.json()["voice_banned"], case["voice_banned"])
 
-    def mock_metricity_user(self, joined_at, total_messages, total_blocks):
+    def test_metricity_review_data(self):
+        # Given
+        joined_at = "foo"
+        total_messages = 10
+        total_blocks = 1
+        channel_activity = [["bar", 4], ["buzz", 6]]
```

Comment:
There should be subtests to test each case in the query (i.e. you need to test off topic and help channels too).